style.css

```diff
@@ -0,0 +1,63 @@
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+}
+
+#hamburger {
+  margin-top: 15px;
+  margin-right: 16px;
+}
+
+body {
+  margin: 0%;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  height: 600px;
+  background-image: url(/images/image_geometry.svg);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-position: center;
+  background-position-y: 100%;
+}
+
```

Comment:
- [x] Please form the project requirement you should stick to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=70%3A1).
  - You must stick to the design as much as possible
    | Your Design (No hamburger and design doesn't match) | Figma Design(With hamburger) |
    | :---:   | :---: | 
    | ![Screenshot from 2022-10-26 18-12-18](https://user-images.githubusercontent.com/92303891/198103975-1b6e938d-b01b-4cbb-867b-d69ca758d98a.png) | ![Screenshot fro](https://user-images.githubusercontent.com/92303891/197907510-54c49e15-e4e7-4d33-8fd8-df0fc15a2272.png) |

Reply:
![image](https://user-images.githubusercontent.com/109642972/198109742-aa6b9973-4c57-4a03-a889-bcb48d6bff04.png)
Hi, I added a screenshot because I don't know if the problem you are mentioning only happened to you. As you can see everything is working.